Dockerfile.android

```diff
@@ -1,9 +1,9 @@
 ARG img_version
 FROM godot-mono:${img_version}
-
 ARG mono_version
 
-RUN if [ -z "${mono_version}" ]; then echo -e "\n\nArgument mono_version is mandatory!\n\n"; exit 1; fi && \
+SHELL ["/bin/bash", "-o", "pipefail", "-c"]
```

Comment:
hadolint says use this line for any run statement that has a pipe in it so the docker will fail if either side of the pipe fails.